tests/performance_tests/performance_test_basic.py

```diff
@@ -209,7 +210,11 @@ def runTpsTest(self) -> bool:
             ])
 
         # Get stats after transaction generation stops
-        self.data.ceaseBlock = self.waitForEmptyBlocks(self.validationNode, self.emptyBlockGoal) - self.emptyBlockGoal + 1
+        trxSent = {}
+        log_reader.scrapeTrxGenTrxSentDataLogs(trxSent, self.trxGenLogDirPath, self.quiet)
```

Comment:
Should we try to find a way to only do `scrapeTrxGenTrxSentDataLogs` one time during the test and reuse the `trxSent` list?  It is also performed in `log_reader.calcAndReport`.  I guess we would still need to make a copy to pass into `waitForTransactionsInBlockRange`, which may mean the overhead of copying and plumbing it through to `calcAndReport` may not be worth it and a second scrape isn't too big of a performance hit.